utils/mvcli.go

```diff
@@ -15,25 +15,9 @@ import (
 
 const mvcliBin = "/usr/bin/mvcli"
 
-var validInfoTypes = map[string]string{
-	"hba": "Storage Controllers",
-	"pd":  "Physical Disks",
-	"vd":  "Virtual Disks",
-}
-
-var validRaidModes = map[string]string{
-	"0":      "Striping (RAID-0)",
-	"1":      "Mirroring (RAID-1)",
-	"10":     "Mirrored Stripe (RAID-10)",
-	"5":      "RAID-5",
-	"1e":     "RAID-1e",
-	"hc":     "hc",
-	"hs":     "hs",
-	"hybrid": "hybrid",
-}
-
+var validRaidModes = []string{"0", "1", "10", "5", "1e", "hc", "hs", "hybrid"}
```

Comment:
The previous map that described each of RAID levels was useful to know what these identifiers are,
suggestion: leave a comment above the Modes, Types with what they mean.

Should this be called `validRaidLevels` instead ?
```
// 0: Striping (RAID-0)
// 1: Mirroring (RAID-1)
```

Reply:
Level.. Mode...  Type...  depends on who you ask, I suppose.   CreateVirtualDisk() and others currently expect RaidMode.

Some of these valid (by marvell standards) 'modes' are probably not 'standard' industry wide (like RAID-1, 5, 10, etc are) so I wasn't thinking we'd 'expose' those at a higher level, which is why I felt it ok to ditch the map and simplify things.  I could add comments, I suppose.

Reply:
A few comments for the modes would suffice